.i18nrc.json

```diff
@@ -6,6 +6,7 @@
     "kbn": "src/core_plugins/kibana",
     "kbnVislibVisTypes": "src/core_plugins/kbn_vislib_vis_types",
     "markdownVis": "src/core_plugins/markdown_vis",
+    "metrics": "src/core_plugins/metrics",
```

Comment:
Could we please rename the id prefix to `tsvb`, since that's the common abbreviation we use for the visual builder.